opengrok-indexer/src/main/java/org/opengrok/indexer/index/Indexer.java

```diff
@@ -432,7 +432,18 @@ public static void main(String[] argv) {
 
         searchPaths.clear();
 
-        // Limit usage lines to 72 characters for concise formatting.
+        /*
+         * FOR CONCISE FORMATTING, LIMIT USAGE DESCRIPTION LINES TO
+         *
+         *    8888888888  .d8888b.                888
+         *          d88P d88P  Y88b               888
+         *         d88P         888               888
+         *        d88P        .d88P       .d8888b 88888b.   8888b.  888d888 .d8888b
+         *     88888888   .od888P"       d88P"    888 "88b     "88b 888P"   88K
+         *      d88P     d88P"           888      888  888 .d888888 888     "Y8888b.
+         *     d88P      888"            Y88b.    888  888 888  888 888          X88
+         *    d88P       888888888        "Y8888P 888  888 "Y888888 888      88888P'
+         */
```

Comment:
:-D

I wonder if there is a way how to enforce via test case.

Reply:
The parser definition and `helpUsage` is encapsulated in `Indexer` and not available for testing easily I think.

Reply:
will be fixed via #3968